### DISCUSSION TITLE
Umami Roadmap

### DISCUSSION THREAD

**Discussion by briancao** [umami-software/umami#1698] [General]
🚀  We've completed our roadmap for 2023 and it's now available to the public.

https://trello.com/b/LIIz6ypc/umami-2023-roadmap

Notably, our primary focus is releasing v2 of Umami Analytics, which includes some well sought after features by the community.
 ⭐ A redesign of the dashboard.
 ⭐ Teams and a UI to manage those Teams.
 ⭐ A refactor of our UI components to use our new 'React Basics' library.
 ⭐ A redesign of the Admin UI.
 ⭐ v1 to v2 migration documentation and scripts.

**Comment by TdhPerso14** [comment]
Super ! I can't wait to use these new functions and more the localization of regions and cities

**Comment by prasannamestha** [comment]
Nice. Maybe we could add user journeys/flow?

**Reply by briancao** [reply]
We have journey/flow labeled as funnels, expected in Q2 2023.

**Reply by kkarimi** [reply]
Any news on the funnel feature? Thanks @briancao

**Reply by briancao** [reply]
The work on this is currently in-progress.  Looking like at least another week.
